src/components/app/ListPage.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { getDevilsLettuce } from '../../services/my-api';
+
+export default class ListPage extends Component {
+  state = {
+    strains: []
+  }
+  
+    componentDidMount = async() => {
+      const strainObj = await getDevilsLettuce();
+      const strain = Object.keys(strainObj);
+    
+      this.setState({ strains: strain });
+    }
+
+    render() {
+      const strainLinks = this.state.strains.map(strain => (
+        <li key={strain}>
+          <a href={`/${strain.id}`}>
```

Comment:
You should use the <Link> component built into react-router-dom instead of <a> tags.